cmd/repository/image_functions.go

```diff
@@ -255,6 +255,22 @@ func GetUntaggedManifests(ctx context.Context, poolSize int, acrClient api.AcrCL
 			}
 
 			// _____MANIFEST IS UNTAGGED BUT MAY BE PROTECTED_____
+			if deleteCutoff != nil {
+				if manifest.LastUpdateTime == nil {
+					fmt.Printf("Skipping manifest %s because the last update time is unavailable\n", *manifest.Digest)
+					continue
+				}
+
+				lastUpdateTime, err := time.Parse(time.RFC3339Nano, *manifest.LastUpdateTime)
+				if err != nil {
+					return nil, err
+				}
+
+				if lastUpdateTime.After(*deleteCutoff) {
+					continue
```

Comment:
We also need to protect possible children here so it might be best to combine with the property above